client/src/widgets/layout/sidebar/ColapsibleMenu.tsx

```
@@ -1,74 +1,63 @@
-        <TooltipProvider delayDuration={150}>
-          <Tooltip>
-            <TooltipTrigger asChild>{header}</TooltipTrigger>
-            <TooltipContent side="right" className="bg-background border shadow-md">
-              {title}
-            </TooltipContent>
-          </Tooltip>
-        </TooltipProvider>
-      ) : (
-        header
-      )}
+      <div className={getCollapsibleHeaderClasses()} onClick={toggleOpen}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)